common/openapi.yml

```diff
@@ -180,36 +198,36 @@ components:
           type: string
           description: Indicates what type of parameter declaration this is.
           required: true
-          enum: [ literal, entity_lookup ]
-          example: entity_lookup
+          enum: [ literal, record_lookup ]
+          example: record_lookup
       oneOf:
-        - $ref: '#/components/schemas/ParameterDefinitionEntityLookup'
+        - $ref: '#/components/schemas/ParameterDefinitionRecordLookup'
         - $ref: '#/components/schemas/ParameterDefinitionLiteralValue'
```

Comment:
Just a heads up, this and above conversion of `entity_lookup` to `record_lookup` is also being done in PR #44 . Depending who merges first there might be merge conflicts.